config/locales/active_admin.zh-CN.yml

```diff
@@ -0,0 +1,645 @@
+        link1: "注销"
+        link1_text: "在后端编辑文章"
+        link2: "将文章设为离线"
+        confirm2: "设置页面脱机？"
+        link3: "将文章设为离线"
+        confirm3: "设置页面为在线？"
+    sessions:
+      login:
+        link1: "注销"
+        label_email: "电子邮件"
+        label_password: "密码"
+        submit: "登录"
+    register:
+      label_email: "电子邮件*"
+      label_username: '用户名 *'
+      label_password: '密码 *'
+      label_pass_confirmation: '确认密码 *'
+      div: "我接受AGB和隐私条款*"
+      submit: "注册"
+  active_admin:
+    redirector:
+      as: "重定向"
+    article_url:
+      rewrite_urls: "重新生成所有的网址"
+      batch_action:
+        rewrite_urls: "重新生成网址"
+        flash:
+          rewrite_urls: "已重新生成网址"
+      as: "文章网址"
+    dashboard: "仪表板"
+    dashboard_welcome:
+      welcome: "欢迎使用Active Admin。 这是标准的仪表板页面。" 
+      call_to_action: "请参阅'app/admin/dashboards.rb'以添加新的仪表板元素。"
+    view: "显示"
+    devise:
+      login:
+        title: "- 管理"
+        remember_me: "记住我"
+      links:
+        sign_up: "注册"
+        forgot_your_password: "忘记密码？"
+        resend_unlock_instructions: "发送解锁指南电子邮件"
+      unlock:
+        title: "发送解锁指南的电子邮件"
+        submit: "现在发送"
+    edit: "编辑"
+    delete: "删除"
+    delete_confirmation: "你真的想删除这个元素？"
+    new_model: "新 %{model}"
+    create_model: "创建 %{model}"
+    edit_model: "编辑 %{model}"
+    delete_model: "删除 %{model}"
+    update_model: "更新 %{model}"
+    details: "%{model} 详细信息"
+    cancel: "取消"
+    empty: "空"
+    previous: "返回"
+    next: "下一个"
+    download: "下载："
+    has_many_new: "添加 %{model}"
+    has_many_delete: "删除"
+    filter: "过滤器"
+    clear_filters: "删除过滤器"
+    search_field: "搜索%{field}"
+    equal_to: "等于"
+    greater_than: "多于"
+    less_than: "少于"
+    main_content: "请实现%{model}#针对内容视图的主要内容"
+    logout: "注销"
+    sidebars:
+      submit_widget: "保存选择"
+      submit_layout: "提交布局"
+      widgets_options: "小工具选项"
+      startpage_options: "起始页选项"
+      filters: "过滤器"
+      layout: "文章类型和布局"
+      image_module: "图片库"
+      image_formates: "图片格式"
+      overview: "概览"
+      menue_options: "菜单选项"
+      link_checker: "确认链接"
+    per_page: "每页的条目数"
+    pagination:
+      empty: "找不到%{model}"
+      one: "显示 <b>1个</b> %{model}"
+      one_page: "显示 <b>所有 %{n}个</b> %{model}"
+      multiple: "显示 %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b> 共 <b>%{total}</b>"
+      multiple_without_total: "显示 %{model} <b>%{from}&nbsp;-&nbsp;%{to}</b>"
+      per_page: "每页："
+      entry:
+        one: "条目"
+        other: "条目"
+    any: "所有"
+    blank_slate:
+      content: "目前还没有%{resource_name}。"
+      link: "创建"
+    batch_actions:
+      labels:
+        set_article_online: "设置文章为在线"
+        set_article_offline: "设置文章为离线"
+        set_menue_offline: "设置菜单为在线"
+        set_menue_online: "设置菜单为离线"
+        reset_cache: "清空缓存"
+        send_default_conf_mails: "发送确认电子邮件"
+        destroy: "删除"
+      button_label: "批量编辑"
+      delete_confirmation: "你确定要删除这些%{plural_model}吗？撤消选项不可用。" available."
```

Comment:
Something is wrong here!
`available."` should have been removed!
I will fix it on master.

Reply:
--> https://github.com/ikuseiGmbH/Goldencobra/commit/5bc3536a0f8dda8e21e28d311ec9ba7084ae5e8e